Bindings/Java/Matlab/examples/Moco/exampleSquatToStand/exampleIMUTracking/exampleIMUTracking_answers.m

```diff
@@ -0,0 +1,334 @@
+function exampleIMUTracking_answers
+
+%% Part 0: Load the Moco libraries and pre-configured Models.
+addpath('../'); % Add the directory above to access mocoPlotTrajectory.m
+import org.opensim.modeling.*;
+
+% These models are provided for you (i.e., they are not part of Moco).
+model = getTorqueDrivenModel();
+% model = getMuscleDrivenModel();
```

Comment:
The name is a bit generic (TorqueDrivenModel of what?) I'd suggest we pinpoint name, tie to example/context